### PR TITLE
Update AudioAnalyzer.yaml

### PR DESCRIPTION
**Pull request by rbxphogen**
## Changes

Add a note about AudioAnalyzer:GetSpectrum returning an empty array.

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/reference/engine/classes/AudioAnalyzer.yaml [hunk 1]
```diff
@@ -117,6 +117,7 @@ methods:
       Returns the frequency spectrum of the last audio buffer, as an array of
       numbers. The elements of the array are root-mean-square volume levels,
       evenly spaced from 0 hertz to 24,000 hertz.
+      However, if any of the analyzer's inputs come from an `Class.AudioDeviceInput`, this method returns an empty array.
```
Comment (rbxphogen) [inline content/en-us/reference/engine/classes/AudioAnalyzer.yaml:120]:
```suggestion
      If any of the analyzer's inputs come from an `Class.AudioDeviceInput`, this method returns an empty array.
```

File: content/en-us/reference/engine/classes/AudioAnalyzer.yaml [hunk 2]
```diff
@@ -117,6 +117,7 @@ methods:
       Returns the frequency spectrum of the last audio buffer, as an array of
       numbers. The elements of the array are root-mean-square volume levels,
       evenly spaced from 0 hertz to 24,000 hertz.
+      If any of the analyzer's inputs come from an `Class.AudioDeviceInput`, this method returns an empty array.
```
Comment (matthewzhang-rbx) [inline content/en-us/reference/engine/classes/AudioAnalyzer.yaml:120]: We also have this disabled for `AudioEcho`, right? Do we need to mention that as well

**Review by matthewzhang-rbx — APPROVED** [review]
Either way, seems fine

**Review by aetter — APPROVED** [review]

File: content/en-us/reference/engine/classes/AudioAnalyzer.yaml (hunk 2 above)
Comment (rbxphogen) [reply content/en-us/reference/engine/classes/AudioAnalyzer.yaml:120]: All the exceptions are logically downstream from AudioDeviceInput

Comment (matthewzhang-rbx) [reply content/en-us/reference/engine/classes/AudioAnalyzer.yaml:120]: I think the relationship is less obvious if it's an AudioEcho or AudioReverb that has never been wired to an AudioDeviceInput before

**Pull request merged**